lib/prawn/embedded_files/filespec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Prawn
+  module EmbeddedFiles
+    class Filespec
+      attr_reader :file_name
+
+      def initialize(file, options = {})
+        name = options[:name] || file.chksum
+
+        @file_name = PDF::Core::LiteralString.new(name)
+
+        if options[:description]
+          desc_str = options[:description].to_s
+          @description = PDF::Core::LiteralString.new(desc_str)
+        end
+
+        @hidden = !options[:hidden].nil? && options[:hidden] == true
```

Comment:
@gettalong I simplified the expression to avoid the "not not". Although, I wouldn't like to use just `options[:hidden]` as a condition because any object (e.g. `'false'`) will be `true`. So I thought it would be more restrictive to require specifically `true` as the option's value

Reply:
Checking for `!options[:hidden].nil?` is avoidable, so I removed too

Reply:
Any value other than `nil` and `false` are `true`. So if you pass in a string, it is a truthy value. Generally, if you mis-use an API you need to expect wrong outcomes. Therefore I wouldn't check for `== true` but for a truthy value.

Reply:
Ok, sounds fair